src/components/Navbar/navbar.css

```diff
@@ -0,0 +1,18 @@
+@import url("../../root.css");
+html {
+  font-size: 10px;
+}
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+nav {
+  background-color: var(--primary-color);
+  padding: 2rem 3rem 2rem 2rem;
```

Comment:
if you are using your own CL, then you must have utility classes for spacing.
_Try to use that._